ccd-definition/CaseField/CaseField-HWF-R2-CUI-nonprod.json

```diff
@@ -0,0 +1,26 @@
+[
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "caseTitleWithHWFFeeNumber",
+    "Label": "### ${helpWithFees.helpWithFeesReferenceNumber} ${applicant1.partyName} Vs ${respondent1.partyName}",
```

Comment:
Use `<h2 class=\"govuk-heading-1\">  ${helpWithFees.helpWithFeesReferenceNumber} ${applicant1.partyName} Vs ${respondent1.partyName}</h2>"` to be consistent with all the views. Same with caseTitleWithHWFHearingNumber

Reply:
updated